### PR TITLE
Steps cleanup

### PR DESCRIPTION
**Pull request by rlespinasse**
## Cleanup labs

- [x] syntax lab 1
- [x] syntax lab 2
- [x] interactions lab 1
- [x] interactions lab 2
- [x] testing lab 1
- [x] testing lab 2 (jest have been removed and workflow is now lab2)
- reuse lab 1 (next PR)
- lifecycle lab 1 (next PR)
- lifecycle lab 2 (next PR)

Some labs are too hard to test automatically, but #27 exists so we can try later to do it

### CODE REVIEW

File: docs/markdown/30-testing/99-exercices.md [hunk 1]
```diff
@@ -1,39 +1,25 @@
 <!-- .slide: class="exercice" -->
 
-# Bats
+# using Bats
```
Comment (ThibDujardin) [inline docs/markdown/30-testing/99-exercices.md:3]:
```suggestion
# Using Bats
```

File: docs/markdown/30-testing/99-exercices.md [hunk 2]
```diff
@@ -1,39 +1,25 @@
...
+Test **shell** scripts using **bats**.
 
 ### [Start](https://github.com/sfeir-open-source/sfeir-school-github-action-dev/tree/v1/steps/30-testing-lab1-action-shell-testing)
 
 ##--##
 
 <!-- .slide: class="exercice" -->
 
-# Jest
+# using Workflow
```
Comment (ThibDujardin) [inline docs/markdown/30-testing/99-exercices.md:17]:
```suggestion
# Using Workflow
```

File: steps/20-interactions-lab1-warning/README.md [hunk 3]
```diff
@@ -9,14 +9,13 @@
...
 
-- Create a repository
-- Add the file `testing-workflow.yml` which will serve you to test the action
-- Add JavaScript action file to start the lab
-  - index.js
-  - package.json
-  - action.yaml
+Create a repository with the content of this folder.
+
+`testing-workflow.yaml` will run and do **NOT** print any warning (on the `tests/package.json` file)
```
Comment (ThibDujardin) [inline steps/20-interactions-lab1-warning/README.md:16]:
```suggestion
`testing-workflow.yaml` will run and do **NOT** print any warning (on the `tests/package.json` file).
```

**Review by ThibDujardin — CHANGES_REQUESTED** [review]

File: steps/30-testing-lab1-action-shell-testing-solution/.github/workflows/testing-workflow.yaml [hunk 4]
```diff
@@ -12,5 +10,5 @@ jobs:
 
       - name: bats test
         run: |
-          ${GITHUB_WORKSPACE}/test/bats/bin/bats ${GITHUB_WORKSPACE}/steps/30-testing-lab1-action-shell-testing-solution/test-labs.bats
+          ${GITHUB_WORKSPACE}/tests/bats/bin/bats ${GITHUB_WORKSPACE}/tests/lab-tests.bats
```
[the remark this answers is not in the document]
Comment (rlespinasse) [reply steps/30-testing-lab1-action-shell-testing-solution/.github/workflows/testing-workflow.yaml:13]: this file have move (by git) as solution workflow, not the previous workflow who run automatically, see `.github/workflows/labs.yaml` for the tests run.

**Review by ThibDujardin — APPROVED** [review]

**Pull request merged**